src/main/java/com/internet/shop/controllers/orders/GetLoggedUserOrdersController.java

```diff
@@ -0,0 +1,33 @@
+package com.internet.shop.controllers.orders;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.model.Order;
+import com.internet.shop.service.OrderService;
+import com.internet.shop.service.UserService;
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/users/orders")
+public class GetLoggedUserOrdersController extends HttpServlet {
```

Comment:
this controller should have been in the previous PR

Reply:
I did it few PRs ago. But As I now understand, I did it in some another way. In my project there are links on the allUsers page and you can see any user orders, not only logged on. So I moved it to "admin" part with allUsers page and added one more controller which shows only logged user orders. But it uses the same jsp as previous. If I should refactor something, please tell me, because now everything is working good.